velox/core/SimpleFunctionMetadata.h

```diff
@@ -518,7 +519,7 @@ class SimpleFunctionMetadata : public ISimpleFunctionMetadata {
   std::string helpMessage(const std::string& name) const final {
     // return fmt::format("{}({})", name, signature_->toString());
     std::string s{name};
-    s.append("(");
+    s.append("\nSignature argument types:\n");
```

Comment:
Why is this change? The comment on L520 needs updating.

Can we share an example of the new message?

Reply:
Yes, in a separate PR https://github.com/facebookincubator/velox/pull/10821, you notice it after then.